### PR TITLE
feat: use Helm for installing Keptn

### PR DESCRIPTION
**Pull request by pchila**
Signed-off-by: Paolo Chila <paolo.chila@dynatrace.com>

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -2,12 +2,22 @@ name: Install/Uninstall Keptn
...
+    description: Helm values to set during install passed as a YAML multiline string
+    required: false
+    default: |-
+      control-plane:
+        apiGatewayNginx:
+          type: LoadBalancer
+      continuous-delivery:
+        enabled: true
+  KEPTN_NAMESPACE:
+    description: Kubernetes namespace where Kptn will be installed
```
Comment (Raffy23) [inline action.yaml]: I think that could be a typo
```suggestion
    description: Kubernetes namespace where Keptn will be installed
```

**Review by Raffy23 — APPROVED** [review]
LGTM

**Comment by pchila** [comment]
JES integration test run using the new action: https://github.com/keptn-contrib/job-executor-service/runs/7092707117

**Review by Raffy23 — APPROVED** [review]

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**